src/staging/staging_main.py

```diff
@@ -397,6 +399,28 @@ def run_staging(
         manual_outliers = None
         StagingMainLogger.info("Loading of Manual Outlier File skipped")
 
+    # Get the latest manual trim file
+    manual_trim_path = paths["manual_imp_trim_path"]
+    latest_man_file = list_files(manual_trim_path, ext="csv", order="newest")[0]
```

Comment:
I think this is nice, but would probs be safer (and easier) just directly specify which trimming file to use.